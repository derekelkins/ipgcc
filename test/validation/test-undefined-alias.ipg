S -> A
     for i = 0 to 3 do B[i, i+1]
     { a = A@10(1).y }
     { b = B@15.y }
     repeat D.x
     repeat E.x until F
     { s = D@10.START }
     { e = E@10.END }
     { e = F@10.END };

A -> "a" { y = "a" };
B -> "b" { y = "b" };
D -> "d" { x = "d" };
E -> "e" { x = "e" };
F -> "f";
