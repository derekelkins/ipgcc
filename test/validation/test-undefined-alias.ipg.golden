Alias A@10 in rule S is undefined
Alias B@15 in rule S is undefined
Alias D@10 in rule S is undefined
Alias E@10 in rule S is undefined
Alias F@10 in rule S is undefined